pycmr/granule.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from requests import get
+import urllib
```

Comment:
Small nitpick, but the python thing to do here is import specifically what you need as opposed to the entire module. So in this case it would be something like `from urllib.parse import quote`.